cryptography/src/bls12381/dkg/ops.rs

```diff
@@ -51,14 +51,31 @@ pub fn evaluate_all<V: Variant>(polynomial: &poly::Public<V>, n: u32) -> Vec<V::
     evals
 }
 
+/// A verified commitment.
+pub struct VerifiedCommitment<'a, V: Variant> {
+    commitment: &'a poly::Public<V>,
```

Comment:
On second thought, I think it may be cleaner to take ownership of commitment and store verified commitments.